### PR TITLE
Central new network - nz-akl

### PR DESCRIPTION
**Pull request by mjdavidson**
Need to think about how the list of all lines should look, maybe make a tweet about it
I think it should look like this:
-Central
--Frequent
--Peak
--Connector
--Local
-South
--Frequent
...

Also, I think we could use this as an opportunity to use the correct diacritics on Māori names

--this needs to be completed by July 7th--

### CODE REVIEW

**Comment by consindo** [comment]
Also need to update the tests and make sure they pass

**Comment by mjdavidson** [comment]
better late than never.

File: server-worker/lines/nz-akl.js [hunk 1]
```diff
@@ -32,7 +33,7 @@ const getColor = (agency_id, code) => {
   case 'NZBGW': // NZ Bus - Go West
     return '#4CAF50'
 
-  case 'NZBML': // NZ Bus - metrolink
+  case 'NZB' || 'NZBML': // NZ Bus - metrolink
```
Comment (consindo) [inline server-worker/lines/nz-akl.js]: I didn't know this actually worked

File: server-worker/lines/nz-akl.js [hunk 2]
```diff
@@ -375,65 +522,94 @@ const allLines = {
...
   '4': [['Matiatia Wharf', 'Onetangi Direct']],
-} 
+}
 
 let lineOperators = {}
 let lineColors = {}
 
 function cacheOperatorsAndShapes() {
-  let todo = []
+  let routes = []
```
Comment (consindo) [inline server-worker/lines/nz-akl.js:993]: Wow it used to be called todo

File: server-worker/lines/nz-akl.js [hunk 3]
```diff
@@ -152,137 +180,193 @@ const friendlyNames = {
...
+  '66': [['Pt Chevalier Beach', 'Sylvia Park', 'Mt Albert Rd']],
+  '68': [['New Lynn', 'Onehunga']],
+  '70': [['Botany', 'Britomart', 'Ellerslie']],
+  '72C': [['Botany', 'Panmure', 'Chapel Rd']],
+  '72M': [['Botany', 'Panmure', 'Millhouse Dr']],
+  '75': [['Glen Innes', 'Wynyard Quarter', 'Remuera Rd']],
+  '380': [['Onehunga', 'Manukau', 'Airport']],
 
+  // CENTRAL - Connector
+  '22R': [['Rosebank Rd',        'City Centre','New North Rd']],
```
Comment (consindo) [inline server-worker/lines/nz-akl.js]: the indentation is a bit funny

**Review by consindo — CHANGES_REQUESTED** [review]
need to fix some of the weird indentation

File: server-worker/lines/nz-akl.js (hunk 2 above)
Comment (mjdavidson) [reply server-worker/lines/nz-akl.js:993]: ...

**Comment by mjdavidson** [comment]
@consindo  this is done, will restructure the groups in the future

**Review by consindo — APPROVED** [review]
💯

**Pull request merged**